File: git.js
const { exec } = require("child_process");
const commitCount = 1000000;
const pushThreshold = 10000;

const gitPull = () => {
  return exec("git pull", (err, stdout, stderr) => {
    if (err) {
      console.log("🔥 pull error: ", err);
      return;
    }
    console.log(`🚀 : ${stdout}`);
  });
};

const gitCommit = () => {
  return exec(
    'git commit --allow-empty -m "go + git + github = 💥"',
    (err, stdout, stderr) => {
      if (err) {
        console.log("🔥 commit error: ", err);
        return;
      }
      console.log(`🚀 : ${stdout}`);
    }
  );
};

const gitPush = () => {
  return exec("git push", (err, stdout, stderr) => {
    if (err) {
      console.log("🔥 push error: ", err);
      return;
    }

    console.log(`🛬 pushed successfully: ${stdout}`);
  });
};

const run = () => {
  gitPull();

  for (let i = 0; i < commitCount; i++) {
    gitCommit();
    console.log("i", i);

    if (i % pushThreshold === 0) {
      gitPush();
    }
  }
};

run();

File: git.go
package main

import (
    "fmt"
    "os/exec"
)

func main() {
    git := "git"

    arg1 := "commit"
    arg2 := "--allow-empty"
    arg3 := "-m"
    message := "'go + git + github = 💥'"
    commitCount := 10//0000

    for i:=0; i<commitCount ;i++ {

        cmd := exec.Command(git, arg1 , arg2, arg3, message)
        stdout, err := cmd.Output()

        if err != nil {
            fmt.Println("🔥",err.Error())
            return
        }
        fmt.Println("🚀",string(stdout))
    }

    push := "push"
    cmd := exec.Command(git, push)
    stdout, err := cmd.Output()

    if err != nil {
        fmt.Println("🔥",err.Error())
        return
    }
    fmt.Println("🛬",string(stdout))   

}
